commands/houndCommands.mjs

```diff
@@ -68,7 +68,7 @@ export async function summonDog(message) {
   if (quotes.length > 0) {
     return sendMessageWithOptions(message, quotes[Math.floor(Math.random() * quotes.length)]);
   }
-  return null;
+  return false;
```

Comment:
On line 55, we're breaking our consistency by suddenly not using arrow functions. please switch it to `export const summonDog = async (message) =>`